odoo/tools/config.py

```diff
@@ -556,11 +561,22 @@ def _check_upgrades_paths(self, option, opt, value, parser):
         for path in value.split(','):
             path = path.strip()
             res = self._normalize(path)
+            if not res:
+                continue
             if not os.path.isdir(res):
                 raise optparse.OptionValueError("option %s: no such directory: %r" % (opt, path))
-            upgrades_paths.append(res)
+            if not self._is_upgrades_path(res):
+                raise optparse.OptionValueError("option %s:  The upgrades-path %r does not seem to a be a valid Upgrade Directory!" % (opt, path))
```

Comment:
The message is in English, let's use English grammar
```suggestion
                raise optparse.OptionValueError("option %s: the path %r is not a valid upgrade directory" % (opt, path))
```